src/test/java/seedu/momentum/logic/commands/CommandTestUtil.java

```diff
@@ -117,6 +117,21 @@ public static void assertCommandSuccess(Command command, Model actualModel, Comm
         }
     }
 
+    //    /**
+    //     * Executes the given {@code command}, confirms that <br>
+    //     * - the returned {@link CommandResult} matches {@code expectedCommandResult} <br>
+    //     */
+    //    public static void assertUndoCommandSuccess(Command command, Model actualModel,
+    //                                                String expectedMessage) {
+    //        try {
+    //            CommandResult result = command.execute(actualModel);
+    //            CommandResult expectedCommandResult = new CommandResult(expectedMessage);
+    //            assertEquals(result, expectedCommandResult);
+    //        } catch (ParseException | CommandException ce) {
+    //            throw new AssertionError("Execution of command should not fail.", ce);
+    //        }
+    //    }
+
```

Comment:
Is this commented out part needed

Reply:
Uhh it's a draft for UndoCommand, can we try to leave it first?

Reply:
Sure